### ISSUE TITLE
✨ Feature Suggestion | Link independent security audits to tools

### ISSUE THREAD

**Issue by nitrohorse** [privacytoolsIO/privacytools.io#1071]
## Description:

From @Mikaela on Wire's PTIO group: should we list independent security audits either in a separate file like [source_code.md](https://github.com/privacytoolsIO/privacytools.io/blob/master/source_code.md) or together with the tools?

Example: [Keybase's 2019 audit](https://keybase.io/docs-assets/blog/NCC_Group_Keybase_KB2018_Public_Report_2019-02-27_v1.3.pdf) by the [NCC Group](https://www.nccgroup.trust)

**Comment by Mikaela** [comment]
IPFS mirror of the Keybase audit https://ipfs.privacytools.io/ipfs/QmSuz9YGsCuAvFzsqFZoRaQ9ejtgojrmbq7RtQN27qHCu8/NCC_Group_Keybase_KB2018_Public_Report_2019-02-27_v1.3.pdf

**Comment by Mikaela** [comment]
Wire: https://crysp.uwaterloo.ca/opinion/wire/

**Comment by nitrohorse** [comment]
Signal: https://eprint.iacr.org/2016/1013.pdf

**Comment by Mikaela** [comment]
Maybe some of them can be put into related information in the IM page?

**Comment by Mikaela** [comment]
I am going to take part of this as per my previous comment, together with #1137 and #967.

**Comment by nitrohorse** [comment]
> Maybe some of them can be put into related information in the IM page?

Yeah I’m wondering how we should link to these from their listings... as a badge?
